crates/codegen/language/definition/src/model/non_terminals/field.rs

```diff
@@ -15,6 +15,10 @@ pub struct FieldsErrorRecovery {
 pub struct FieldDelimiters {
     pub open: Identifier,
     pub close: Identifier,
+    /// How many tokens have to be matched to trigger the error recovery.
+    /// For ambiguous syntaxes this needs to be set to at least N, where N
+    /// is the token lookahead required to disambiguate the syntax.
```

Comment:
thinking out loud:

I wonder if it is more idiomatic to use `usize` instead if `u8` for counting/lengths/indices/etc... It would also relieve us from worrying about or silencing `clippy` lints because of truncation/casting.

Reply:
In a very pure sense yes, but we're talking about grammar token lookahead necessary to disambiguate; if someone specifies more than, let's say, 5 I'd be very surprised and if someone needs to specify >= 256 they have a bigger problem as our product won't really be usable due to the sheer amount of backtracking required and performed.

As for truncation or casting, the new function `matches_at_least_n_tokens` that uses the value already accepts `n: u8`, which either comes from the user or uses the default `0u8`, so we're safe on that front.